apps/people/migrations/0004_people_is_founder.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1.5 on 2023-06-19 09:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('people', '0003_people_order'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='people',
+            name='is_founder',
+            field=models.BooleanField(default=True, verbose_name='Is Founding Member'),
```

Comment:
Instead of creating new migrations, let's just update this as well
```suggestion
            field=models.BooleanField(default=False, verbose_name='Is Founding Member'),
```